src/main/java/com/alipay/oceanbase/rpc/util/MonitorUtil.java

```diff
@@ -117,7 +117,7 @@ private static String logMessage(String traceId, String database, String tableNa
             endpoint = endpoint.replaceAll(",", "#");
         }
 
-        String argsValue = buildParamsString(Arrays.asList(rowKeys));
+        String argsValue = rowKeys.isEmpty() ? "" : buildParamsString(Arrays.asList(rowKeys));
```

Comment:
```suggestion
        String argsValue = rowKeys.isEmpty() ? "rowkeys: []" : "rowkeys + "buildParamsString(Arrays.asList(rowKeys));
```
May have some problems here, but I guess we should tell user what is ""

Reply:
edit